content/en/docs/concepts/overview/components.md

```diff
@@ -14,140 +13,73 @@ card:
+<dl>
+  <dt>kube-apiserver</dt>
+  <dd>The API server that exposes the Kubernetes API</dd>
 
-{{< glossary_definition term_id="etcd" length="all" >}}
+  <dt>etcd</dt>
+  <dd>Consistent and highly-available key value store for all cluster data</dd>
 
-### kube-scheduler
+  <dt>kube-scheduler</dt>
+  <dd>Assigns Pods to Nodes</dd>
 
-{{< glossary_definition term_id="kube-scheduler" length="all" >}}
+  <dt>kube-controller-manager</dt>
+  <dd>Runs controller processes</dd>
 
-### kube-controller-manager
+  <dt>cloud-controller-manager</dt>
+  <dd>Integrates with underlying cloud providers</dd>
+</dl>
 
-{{< glossary_definition term_id="kube-controller-manager" length="all" >}}
+### Node Components
 
-There are many different types of controllers. Some examples of them are:
+Run on every node, maintaining running pods and providing the Kubernetes runtime environment:
 
-  * Node controller: Responsible for noticing and responding when nodes go down.
-  * Job controller: Watches for Job objects that represent one-off tasks, then creates
-    Pods to run those tasks to completion.
-  * EndpointSlice controller: Populates EndpointSlice objects (to provide a link between Services and Pods).
-  * ServiceAccount controller: Create default ServiceAccounts for new namespaces.
+<dl>
+  <dt>kubelet</dt>
+  <dd>Ensures that containers are running in a Pod</dd>
 
-The above is not an exhaustive list.
+  <dt>kube-proxy</dt>
+  <dd>Maintains network rules on nodes</dd>
 
-### cloud-controller-manager
-
-{{< glossary_definition term_id="cloud-controller-manager" length="short" >}}
-
-The cloud-controller-manager only runs controllers that are specific to your cloud provider.
-If you are running Kubernetes on your own premises, or in a learning environment inside your
-own PC, the cluster does not have a cloud controller manager.
-
-As with the kube-controller-manager, the cloud-controller-manager combines several logically
-independent control loops into a single binary that you run as a single process. You can
-scale horizontally (run more than one copy) to improve performance or to help tolerate failures.
-
-The following controllers can have cloud provider dependencies:
-
-  * Node controller: For checking the cloud provider to determine if a node has been deleted in the cloud after it stops responding
-  * Route controller: For setting up routes in the underlying cloud infrastructure
-  * Service controller: For creating, updating and deleting cloud provider load balancers
-
-## Node Components
-
-Node components run on every node, maintaining running pods and providing the Kubernetes runtime environment.
-
-### kubelet
-
-{{< glossary_definition term_id="kubelet" length="all" >}}
-
-### kube-proxy
-
-{{< glossary_definition term_id="kube-proxy" length="all" >}}
-
-### Container runtime
-
-{{< glossary_definition term_id="container-runtime" length="all" >}}
+  <dt>Container runtime</dt>
+  <dd>Software responsible for running containers</dd>
+</dl>
 
 ## Addons
```

Comment:
```suggestion
## Add-ons
```
I don't know why this suggestion box extends into the two editor layouts.

Reply:
This suggestion is not part of the change that the PR description is summarizing, so @robert-cronin you are welcome to skip it.

Reply:
@network-charles I'll skip this change to maintain consistency with current docs as discussed above. Thanks for flagging it.

Reply:
Ok